### PR TITLE
Move more CI to php7.2 as minimum

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Comment by J0WI** [comment]
Context: https://www.php.net/supported-versions.php

**Review by ChristophWurst — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -2334,34 +2252,34 @@ trigger:
...
-  image: morrisjobke/redis-cluster
-
-trigger:
-  branch:
-    - master
-    - stable*
-  event:
-    - pull_request
-    - push
+#---
```
Comment (juliushaertl) [inline .drone.yml]: Any specific reason why that is disabled? :wink:

Comment (rullzer) [reply .drone.yml]: It fails hard on 7.2
But it also alrady failed on 7.1 but there it was just ignored by phpunit for some reason.

Comment (juliushaertl) [reply .drone.yml]: Ok then fine by me

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**